akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +58,11 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    CompletionStage<Done> ignoreDataBytesJ(Materializer materializer);
```

Comment:
That's a different ticket, I'm trying to ease the team into the domain here.
The next ticket is issuing warnings and auto-draining features which is https://github.com/akka/akka/issues/18716 and https://github.com/akka/akka/issues/18540

So here we *really* just need the names.